modules/serverpod_auth/serverpod_auth_server/lib/src/business/user_authentication.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_auth_server/module.dart';
+import 'package:serverpod_auth_server/src/business/authentication_util.dart';
+import 'package:serverpod_shared/serverpod_shared.dart';
+
+/// Collects methods for authenticating users.
+class UserAuthentication {
+  /// Signs in an user to the server. The user should have been authenticated
+  /// before signing them in. Send the AuthKey.id and key to the client and
+  /// use that to authenticate in future calls. In most cases, it's more
+  /// convenient to use the serverpod_auth module for authentication.
```

Comment:
Suggestion: We should probably update this description, right?

> _In most cases, it's more convenient to use the serverpod_auth module for authentication._

This refers to the providers in the auth module, right?